services/management/ia/templates_service.go

```diff
@@ -40,9 +42,21 @@ import (
 	"github.com/percona/pmm-managed/data"
 	"github.com/percona/pmm-managed/models"
 	"github.com/percona/pmm-managed/utils/dir"
+	"github.com/percona/pmm-managed/utils/envvars"
+	"github.com/percona/pmm-managed/utils/saasdial"
+	"github.com/percona/pmm-managed/utils/signatures"
 )
 
-const templatesDir = "/srv/ia/templates"
+const (
+	templatesDir = "/srv/ia/templates"
+	envPublicKey = "PERCONA_TEST_CHECKS_PUBLIC_KEY"
+)
+
+var defaultPublicKeys = []string{
+	"RWTfyQTP3R7VzZggYY7dzuCbuCQWqTiGCqOvWRRAMVEiw0eSxHMVBBE5", // PMM 2.6
+	"RWRxgu1w3alvJsQf+sHVUYiF6guAdEsBWXDe8jHZuB9dXVE9b5vw7ONM", // PMM 2.12
+	"RWTHhufOlJ38dWt+DrprOg702YvZgqQJsx1XKfzF+MaB/pe9eCJgKkiF", // PMM 2.17
+}
```

Comment:
Now we have this in two different services. Probably better to have another service that keeps public keys and can verify signatures.

Reply:
i can just move it to utils instead of making a new service

Reply:
Ok, let's try to move it to the `utils`.

Reply:
done, the keys are now in a single file in utils

Reply:
@artemgavrilov PTAL